pkg/userdata/coreos/userdata.go

```diff
@@ -173,8 +173,9 @@ systemd:
         After=docker.service
         [Service]
         TimeoutStartSec=5min
-        Environment=KUBELET_IMAGE_TAG={{ .HyperkubeImageTag }}
+        Environment=KUBELET_IMAGE=docker://k8s.gcr.io/hyperkube-amd64:{{ .HyperkubeImageTag }}
         Environment="RKT_RUN_ARGS=--uuid-file-save=/var/cache/kubelet-pod.uuid \
+          --insecure-options=image \
```

Comment:
why do we need to pass this flag ?

Reply:
Because otherwise `rkt` won't run a docker image: https://coreos.com/rkt/docs/latest/running-docker-images.html